matchzoo/models/__init__.py

```diff
@@ -7,8 +7,9 @@
 from .conv_knrm_model import ConvKNRMModel
 from .duet_model import DUETModel
 from .drmmtks_model import DRMMTKSModel
+from .anmm_model import ANMMModel
 from .drmm import DRMM
 
 import matchzoo
 def list_available():
-    return matchzoo.engine.BaseModel.__subclasses__()
\ No newline at end of file
+    return matchzoo.engine.BaseModel.__subclasses__()
```

Comment:
Don't add this line. #515 has the same change. Both of you should be careful...

Reply:
I will fix this.

Reply:
I didn't add this line. I just added the following line in line 10 .

`from .anmm_model import ANMMModel`